cookbooks/bcpc-hadoop/attributes/default.rb

```diff
@@ -49,6 +49,30 @@
 default["bcpc"]["hadoop"]["hbase"]["superusers"] = ["hbase"]
 # Interval in milli seconds when HBase major compaction need to be run. Disabled by default
 default["bcpc"]["hadoop"]["hbase"]["major_compact"]["time"] = 0
+default["bcpc"]["hadoop"]["hbase"]["bucketcache"]["enabled"] = false
```

Comment:
The number of attributes for `HBase` is large enough to have a `hbase.rb` attributes file just for the HBase settings.